src/components/OSCALCatalogGroups.js

```diff
@@ -0,0 +1,91 @@
+import Box from "@mui/material/Box";
+import List from "@mui/material/List";
+import Paper from "@mui/material/Paper";
+import { styled } from "@mui/material/styles";
+import Tab from "@mui/material/Tab";
+import Tabs from "@mui/material/Tabs";
+import PropTypes from "prop-types";
+import React from "react";
+import { OSCALSection, OSCALSectionHeader } from "../styles/CommonPageStyles";
+import OSCALCatalogGroup from "./OSCALCatalogGroup";
+
+export const OSCALControlList = styled(List)`
+  padding-left: 2em;
+  padding-right: 2em;
+`;
+
+function TabPanel(props) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`vertical-tabpanel-${index}`}
+      aria-labelledby={`vertical-tab-${index}`}
+      {...other}
+    >
+      {value === index && <Box sx={{ p: 3 }}>{children}</Box>}
+    </div>
+  );
+}
```

Comment:
Why not just use https://mui.com/material-ui/api/tab-panel/?

Reply:
Most of the examples https://mui.com/material-ui/react-tabs/ do not use that `tab-panel`. They always use a function like that. I can look into using the already made component though.